app/main.py

```diff
@@ -27,11 +27,15 @@
     collection_of_coins
 ]
 
-sorted_variables = {"mutable": [], "immutable": []}
+mutable = [
+    variable for variable in variables_list
+    if isinstance(variable, (list, dict, set))
+]
+immutable = [
+    variable for variable in variables_list
+    if not isinstance(variable, (list, dict, set))
+]
+
+sorted_variables = {"mutable": mutable, "immutable": immutable}
```

Comment:
You didn't fix previous comment. Let's do it manually, create dictionary with 2 keys: `mutable` and `immutable` and then manually add all mutable/immutable variables. You  can do in the smae method, as example tells you